README.md

```diff
@@ -91,7 +91,7 @@ a) In `appsettings.json` file, configure items under `XRay` key
     "SamplingRuleManifest": "SamplingRules.json",
     "AWSXRayPlugins": "EC2Plugin, ECSPlugin, ElasticBeanstalkPlugin",
     "AwsServiceHandlerManifest": "JSONs\AWSRequestInfo.json",
-    "UseRuntimeErrors":"true",
+    "UseRuntimeErrors":"false",
```

Comment:
Does this config only affect the context missing strategy?

Reply:
`UseRuntimeErrors` is used just for missing contexts. Here are a couple of comments describing `UseRuntimeErrors`

https://github.com/aws/aws-xray-sdk-dotnet/blob/master/sdk/src/Core/Internal/Utils/AppSettings.netcore.cs#L88-L91

https://github.com/aws/aws-xray-sdk-dotnet/blob/master/sdk/src/Core/Internal/Utils/AppSettings.netframework.cs#L88-L91